<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Tests"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.002">
  <properties />
            <testcase name="getTest" time="0.002" classname="weather_zuul.api_v1_forecast_hourly">
                <failure type="org.testng.internal.reflect.MethodMatcherException"
                  message="
Data provider mismatch
Method: getTest([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=3, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=4, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=5, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=6, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=7, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=8, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String)test001,(java.lang.String)WS,(java.lang.String)2019-04-19T08:00:00Z,(java.lang.String)126.14,(java.lang.String)48.17,(java.lang.String)100,(java.lang.String)200]"
              >
<![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: getTest([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=3, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=4, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=5, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=6, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=7, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=8, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String)test001,(java.lang.String)WS,(java.lang.String)2019-04-19T08:00:00Z,(java.lang.String)126.14,(java.lang.String)48.17,(java.lang.String)100,(java.lang.String)200]
  org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:49)
  org.testng.internal.Invoker.injectParameters(Invoker.java:1293)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1187)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:744)
  org.testng.TestRunner.run(TestRunner.java:602)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
  org.testng.TestNG.runSuites(TestNG.java:1144)
  org.testng.TestNG.run(TestNG.java:1115)
  org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
  org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
